static/bidder-info/adhese.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "https://ads-{{.AccountID}}.adhese.com/json"
+endpoint: "https://ads-{{.AccountID}}.adhese.com/openrtb2"
```

Comment:
consider that usage of partial dynamic urls is being discouraged and prebid team in future will be working to avoid usage of dynamic subdomain in endpoint url.
Major concerns with such usage are,
- security concerns
The security aspect is alleviated by using a fixed top level domain. Due to the potential harm to hosts, we are strict in this requirement. We are working towards fixing the few adapters currently in violation.
- connection performance
The connection performance advice is for your benefit. Client specific subdomains prevent Prebid Server from reusing connections across your clients which results in more connections needed for your adapter. The issue gets worse the more successful you are in attracting new clients. Hosts may choose to disable your adapter due to this behavior.

Reply:
Can we address this concern separately from this PR?

Historically we've been always assigning individual domains to each tenant in out system, eg. `ads-demo.adhese.com` for `demo` account. 

To change this approach we'd need to introduce a few architectural changes to our services. I'm happy to plan them in, but we'd rather move forward with the new adapter before the new endpoint approach is implemented.